{
use_imm: true,
#Define number of models
number_of_models: 2,
#Setting Markov transition probabilites
markov_trans_prob_0_0: 0.6,
markov_trans_prob_0_1: 0.4,
markov_trans_prob_1_0: 0.4,
markov_trans_prob_1_1: 0.6,
#Configuration of models used
IMM0_motion_model: ConstantVelocity,
    IMM0_cosxx: 0.4,
    IMM0_cosyy: 0.4,
    IMM0_cosw: 0.7,
    IMM0_cosvxx: 1.0,
    IMM0_cosvyy: 1.0,
    IMM0_use_process_noise: true,
    IMM0_process_noise_level: 0.01,
    IMM0_qsvxx: 0.0064,
    IMM0_qsvyy: 0.0064,
    IMM0_qsxx: 0.0081,
    IMM0_qsyy: 0.0081,
IMM1_motion_model: CoordinatedTurn,
    IMM1_cosxx: 0.7,
    IMM1_cosyy: 0.7,
    IMM1_cosvxx: 2.0,
    IMM1_cosvyy: 2.0,
    IMM1_cosw: 0,
    IMM1_use_process_noise: true,
    IMM1_process_noise_level: 0.1,
    IMM1_qsvxx: 0.0064,
    IMM1_qsvyy: 0.0064,
    IMM1_qsxx: 0.0081,
    IMM1_qsyy: 0.0081,
ct_turn_rate_variance: 3.14,
send_imm_debug_information: true,
use_imm_visualization: true,
}
